src/components/circular-progress/circular-progress.tsx

```diff
@@ -46,6 +46,12 @@ export class CircularProgress {
     @Prop()
     public maxValue: number = PERCENT;
 
+    /**
+     * The prefix which is displayed before the `value`, must be one or two characters long. When the prefix is null, no prefix is used.
```

Comment:
the last sentence is explaining the "obvious", in a developer's way. We don't have to say how the code works, if you know what I mean :)

Reply:
Additionally, it's better to truncate it.